coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/core/ProposalMedusaSection.xml

```diff
@@ -239,15 +239,46 @@
                 <bean parent="Proposal-MedusaSection-Actions" p:render="@{#lp.document.documentNumber ne #np.awardDocument.documentNumber}" p:order="5">
                     <property name="group" ref="Proposal-MedusaSection-AwardSummary-Actions"/>
                 </bean>
+
                 <bean parent="Proposal-Award-Property" p:propertyName="awardNumber" p:order="10" />
-                <bean parent="Proposal-Award-Property" p:propertyName="unitNumber" p:order="20" />
-                <bean parent="Proposal-Award-Property" p:propertyName="awardStatus.description" p:order="30" />
-                <bean parent="Proposal-Award-Property" p:propertyName="ospAdministratorName" p:order="40" />
-                <bean parent="Proposal-Award-Property" p:propertyName="principalInvestigatorName" p:order="50" />
-                <bean parent="Proposal-Award-Property" p:propertyName="sponsorName" p:order="60" />
+                <bean parent="Proposal-Award-Property" p:propertyName="awardType.description" p:order="20" />
+                <bean parent="Proposal-Award-Property" p:propertyName="sponsorAwardNumber" p:order="30" />
+                <bean parent="Proposal-Award-Property" p:propertyName="activityType.description" p:order="40" />
+                <bean parent="Proposal-Award-Property" p:propertyName="accountNumber" p:order="50" />
+                <bean parent="Proposal-Award-Property" p:propertyName="accountTypeDescription" p:order="60" />
+                <bean parent="Proposal-Award-Property" p:propertyName="awardStatus.description" p:order="70" />
+                <bean parent="Proposal-Award-Property" p:propertyName="title" p:order="80" />
+
```

Comment:
I left spaces in here because in the 5.2.1 medusa there's subheaders.  I couldn't get them to work properly.  If anyone has any ideas, let me know.